### ISSUE TITLE
this.get is not a function

### ISSUE THREAD

**Issue by pak11273** [b4dnewz/monitor.io#1]
@b4dnewz I cloned your fork and tried it out. I get a backbone error: this.get is not a function.

**Comment by pak11273** [comment]
I'm using the fork now and got it up and running.  It's awkward that I don't have to use telnet and open up another terminal.  Instead, the interface opens up under the same terminal that I start my server :(.  It does show when sockets are connected but jkhl key movements don't work and messages are not appearing in the monitor.  At least it's a good start.  I hope you continue to improve it.  thanks!

**Comment by b4dnewz** [comment]
hi @pak11273 thanks for interesting in the project! 

I decided to fork the original a couple of months ago in the attempt to make it working again, when I saw it for the first time wasn't working, 

I've tried to understand the logic and adapt the new code to replace __ansi and telnet__ (which was the problem that caused monitor.io to stop working, abandoned project) with a better module called [terminal-kit](https://github.com/cronvel/terminal-kit) trying to keep the same interface and functions.

It's an hard work I've to admit it, because also terminal kit doesn't play well sometimes especially with windows terminals and some other cases but is still better than before and allowed me to make it working again (still buggy).

Unfortunately I forgot to **update the readme** file once I've totally changed the keys and some of the functionalities... to move you should use arrow keys, for now just up n down, this sunday I will update the readme file to be less misleading.

You can try to set the [remote](https://github.com/b4dnewz/monitor.io/blob/master/example/index.js#L6-L10) property to true when you init the monitor options and this will leave the original console output untouched and you will be able (or not depend on the system tty for your terminal) to open the monitor terminal as you expected.


If you have any suggestion, bug report, feature request or whatever feel free to ask and/or open another issue, I hope soon the project will be ready because I need to use it to manage (sometimes) a socket server I own and also for debug and development purposes.

**Comment by pak11273** [comment]
Awesome!  You cleared up a lot of things.  I am also having to manage a socket.io app and I don't have time right now to try and build an admin for it.  I'm sure there are hundreds of others who are looking for this type of solution.

**Comment by b4dnewz** [comment]
yes, I bet this too.. basically as per my experience it is little difficult to debug socketio instances even with the [debuggin scope](https://socket.io/docs/logging-and-debugging/#available-debugging-scopes) server side `DEBUG=* node server.js` or client side `localStorage.debug = '*';` it will be really useful to have an "admin interface" to see what's going on and to interact with all or specific sockets.

That's why when I saw monitor.io I was so excited about it.
